.cookiecutter/includes/requirements/prod.in

```diff
@@ -8,11 +8,9 @@ pyramid-exclog
 pyramid-jinja2
 pyramid-sanity
 pyramid-services
-pyramid-tm
 requests
 whitenoise
 google-auth-oauthlib
 marshmallow
 webargs
 youtube-transcript-api
-zope.sqlalchemy
```

Comment:
`pyramid-tm` and `zope.sqlalchemy` are no longer needed in this include: the cookiecutter nows adds them to `prod.in` for you